### PR TITLE
gh-200. Client Side Logger.

### LINKED ISSUE #200: Provide a Client-Side logger
**Problem:**

No logger for browser

**Solution:**

Provide logger for client side

**Technical Solution:**

We have a pino logger in core, it has implementation for browser need to reseach

- log redux action
- log requests passed from client
- log errors got in componentDidCatch
- need an endpoint to push logs to server ...

### PR DESCRIPTION
**Pull request by yzhbankov**
close #200

### CODE REVIEW

File: apps/web-portal/lib/use-cases/log/LogSave.mjs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {UseCase} from '@zorko-io/util-use-case'
+
+export class LogSave extends UseCase {
+  // eslint-disable-next-line no-unused-vars
+  async run(logs) {
+    console.log('logs ', logs)
```
Comment (nesterone) [inline apps/web-portal/lib/use-cases/log/LogSave.mjs:8]: add todo to wire with 'log' Pino instance

**Review by nesterone — CHANGES_REQUESTED** [review]
Looks, good 

Could you also extract task for testing log endpoint with integration ?

File: apps/web-portal/lib/use-cases/log/LogSave.mjs (hunk 1 above)
Comment (yzhbankov) [reply apps/web-portal/lib/use-cases/log/LogSave.mjs:8]: done

**Comment by nesterone** [comment]
@yzhbankov  plz fix merge convicts

**Review by nesterone — APPROVED** [review]

**Pull request merged**